### ISSUE TITLE
Usage of framework

### ISSUE THREAD

**Issue by k-radlak** [waymo-research/waymo-open-dataset#301]
Hi, 
In the README.md for Real-time Detection Challenges, we can find the following information:

"_As such, users can use arbitrary versions of TensorFlow or PyTorch for their submissions with ease. In particular, we have tested TensorFlow 2.3.0 and 2.4.1, PyTorch 1.3 and 1.7.1, CUDA 10.0 and 11.0, and cuDNN 7 and 8, but other versions, especially older versions, should likely work with little issue. The only required dependency that users must have installed is numpy."

So, it is allowed to use different ML framework than TensorFlow and PyTorch?

**Comment by peisun1115** [comment]
yes, it is allowed as long as you can get it working. All we need is to invoke the docker.
